### PR TITLE
feat: va-topbar

### PR DESCRIPTION
**Pull request by smellyshovel**
:warning: **Important**: make sure to review/merge the [UI-part of this PR](https://github.com/epicmaxco/vuestic-ui/pull/3) as well.

Also make sure to checkout respective branches on *both* repos during testing/reviewing.

### CODE REVIEW

File: src/components/admin/AppLayout.vue [hunk 1]
```diff
@@ -1,19 +1,24 @@
 <template>
```
Comment (asvae) [inline src/components/admin/AppLayout.vue]: image

Comment (asvae) [reply src/components/admin/AppLayout.vue]: Leaps and jumps.

Comment (asvae) [inline src/components/admin/AppLayout.vue]: Hover shouldn't trigger underline.

image

Comment (asvae) [inline src/components/admin/AppLayout.vue]: Click on menu item should close dropdown

image

**Comment by ssemenkoff** [comment]
Fixed described problems. Check this updated [PR](https://github.com/epicmaxco/vuestic-ui/pull/3) in vuestic-ui repository.

**Comment by asvae** [comment]
@ssemenkoff 

Stretch please.
image

**Comment by Kreezag** [comment]
These changes are moved to #691

**Pull request closed without merging**